bin/run

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env node
 
+const { Config } = require('@oclif/config')
```

Comment:
Oh... Isn't this file automatically generated? Are you sure you can monkey-patch it without getting it overwritten on the next iteration?

Reply:
When you type `kourou auth:getCurrentUser`, the custom `command_not_found` hook is executed and we can use the `sdk:query` command with the correct controller and action.

When you type `kourou auth:getCurrentUser --help`, the `@ocliff/plugin-help` take precedence and use the `Config` class from the `@ocliff/config` package and since the command does not exists it displays the `command not found` error.

They cannot overwrite this monkey patch but if they release a new version of `@ocliff/config` then it may cause issues but they don't seems to update this [part of the code](https://github.com/oclif/config/blob/master/src/config.ts#L384) often.

Reply:
👍